hackathon_site/dashboard/frontend/src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -2,12 +2,69 @@ import React from "react";
 import Typography from "@material-ui/core/Typography";
 import Header from "components/general/Header/Header";
 import { hackathonName } from "constants.js";
+import OrderCard from "components/orders/OrderCard/OrderCard";
+import { useDispatch, useSelector } from "react-redux";
+import styles from "./AdminDashboard.module.scss";
+import Grid from "@material-ui/core/Grid";
+import Button from "@material-ui/core/Button";
+import {
+    adminOrderSelectors,
+    getOrdersWithFilters,
+    setFilters,
+} from "../../slices/order/adminOrderSlice";
+import { OrderFilters } from "../../api/types";
 
 const AdminDashboard = () => {
+    const dispatch = useDispatch();
+    const orders = useSelector(adminOrderSelectors.selectAll);
+    const pendingFilter: OrderFilters = {
+        status: ["Submitted"],
+    };
+    dispatch(setFilters(pendingFilter));
+    dispatch(getOrdersWithFilters());
+    const ordersLength = orders.length < 7 ? orders.length : 6;
```

Comment:
Does this order length cover the whole Size of the page? Ideally it should be as close to the Figma design and I believe that essentially covers the width of the webpage. Please use a variable for the size, we typically don't want to be using numeric literals.

Reply:
Yes, this does cover the whole size of the page. I've defined the grid items to be 2 for large:
<img width="175" alt="image" src="https://github.com/ieeeuoft/hackathon-template/assets/54194957/adcf9d64-2d8b-4f61-ada8-b8b0de472d7e">
So a total of 6 items would fit in the 12 size total. 

This is what it looks like with a max of 6:
<img width="1468" alt="image" src="https://github.com/ieeeuoft/hackathon-template/assets/54194957/d3e9fa88-b970-482d-81df-c1c6ae7813ad">

I think if we try adding another one, each card would be too small. So i think this is a good amount. I'll make a constant instead of the numeric literal